lib/driver.h

```diff
@@ -70,6 +71,7 @@ typedef struct _LogDriverPlugin LogDriverPlugin;
 
 struct _LogDriverPlugin
 {
+  SignalSlotConnector *signal_connector;
```

Comment:
Initialise this to NULL in `log_driver_plugin_init_instance` at least, to avoid this containing junk which currently it is

Reply:
Also handle it in free if not null

Reply:
> Initialise this to NULL in `log_driver_plugin_init_instance` at least, to avoid this containing junk which currently it is

upss, I forgot that completely at the end, thanks, fixed.

Reply:
> Also handle it in free if not null

just like for the name member, this class does not own the signal_connector, it is not needed to check or alter it